File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        A = []
        for i in nums1:
            A.append(i)
        for j in nums2:
            A.append(j)
        A.sort()
        p = len(A)
        if (p % 2 == 0):
            return ((float)(A[p/2]+ A[(p/2)-1])/2)
        else:
            return (A[p/2])       

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution 
{
    public double findMedianSortedArrays(int[] nums1, int[] nums2) 
    {
        int m = nums1.length;
        int n = nums2.length;
        int A[] = new int [m + n];
        int c = 0;
        for (int i = 0; i < m; i++)
        {
            A[i] = nums1[i];
            c++;
        }
        for (int i = 0; i < n; i++)
        {
            A[c+i] = nums2[i];
        }
        Arrays.sort(A);
        int p = m + n;
        if (p % 2 == 0)
        {
            return (((double)(A[p/2]+ A[(p/2)-1])/2));
        }
        else
        {
            return (A[p/2]);
        }
    }
}